drivers/kscan/kscan_ite_it8xxx2.c

```diff
@@ -451,18 +446,16 @@ static int kscan_it8xxx2_init(const struct device *dev)
 	 * NOTE: Set input temporarily for gpio_pin_configure(), after that
 	 *       pinmux_pin_set() set to alternate function immediately.
 	 */
-	gpio_pin_configure(config->gpio_dev,
-			   config->alt_list[KSO16].pin,
-			   (GPIO_OPEN_DRAIN | GPIO_INPUT | GPIO_PULL_UP));
-	gpio_pin_configure(config->gpio_dev,
-			   config->alt_list[KSO17].pin,
-			   (GPIO_OPEN_DRAIN | GPIO_INPUT | GPIO_PULL_UP));
-	pinmux_pin_set(config->alt_list[KSO16].pinctrls,
-		       config->alt_list[KSO16].pin,
-		       config->alt_list[KSO16].alt_fun);
-	pinmux_pin_set(config->alt_list[KSO17].pinctrls,
-		       config->alt_list[KSO17].pin,
-		       config->alt_list[KSO17].alt_fun);
+	gpio_pin_configure_dt(&config->kso16_gpios,
+			      (GPIO_OPEN_DRAIN | GPIO_INPUT | GPIO_PULL_UP));
+	gpio_pin_configure_dt(&config->kso17_gpios,
+			      (GPIO_OPEN_DRAIN | GPIO_INPUT | GPIO_PULL_UP));
```

Comment:
`GPIO_OPEN_DRAIN` and `GPIO_PULL_UP` should be in Devicetree GPIO flags cell, I think.

Reply:
Done.